python/lsst/afw/fits.cc

```diff
@@ -22,20 +22,28 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
```

Comment:
In general the template file comments out the operators and stl include lines and we uncomment them as needed. Pim and I have been leaving these in because it is common to require new functionality (like operators) when wrapping packages downstream.